circle.yml

```diff
@@ -230,7 +233,8 @@ jobs:
       - checkout
       - build
       - test
-      - benchmark
+      - benchmark:
+          min_time: "0.01"
```

Comment:
What is this doing?

Reply:
Normally benchmarks run for at least 0.5 s. But in sanitizers run we only want to find bugs in benchmarking code. With this they run only for 10 ms and whole job finishes faster. Except for the ecpairing where single iteration takes ~30 s.